packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -154,18 +155,33 @@ function wrap(middleware, handleNoSuchObject = false) {
 async function _getDashboardStats(app) {
   const dashboard = {}
 
+  let hvSupportedVersions
+  try {
+    hvSupportedVersions = await app.apiMethods['xoa.getHVSupportedVersions']()
+  } catch (e) {
+    hvSupportedVersions = {
+      'Citrix Hypervisor': '>=8.2.0',
+      'XCP-ng': '>=8.2.0',
+    }
```

Comment:
We should not duplicate this info, simply ignore this value if unknown.

Also, maybe we should log the error to understand what went wrong.